mne/io/curry/curry.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: UTF-8 -*-
+#
+# Authors: Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+#
+#
+# License: BSD (3-clause)
+
+import os
+import re
+import numpy as np
+import mne
+
+from ..base import BaseRaw, _check_update_montage
+from ..meas_info import create_info
+from ..utils import _read_segments_file, warn, _find_channels, _create_chs
+
+
+def _read_curry_events(fname_base, curry_vers):
+    """
+    Read curry files and convert the data for mne.
+    Inspired by Matt Pontifex' EEGlab loadcurry() extension.
+
+    :param full_fname:
+    :return:
+    """
+
+    #####################################
+
+    EVENT_FILE_EXTENSION = {7: ['.cef', '.ceo'], 8: ['.cdt.cef', '.cdt.ceo']}
+
+    event_file = None
+    for ext in EVENT_FILE_EXTENSION[curry_vers]:
+        if os.path.isfile(fname_base + ext):
+            event_file = fname_base + ext
+
+    if event_file is not None:
+        curry_events = []
+        save_events = False
+        with open(event_file) as fid:
+            for line in fid:
+
+                if "NUMBER_LIST END_LIST" in line:
+                    save_events = False
+
+                if save_events:
+                    curry_events.append(line.split("\t"))
+
+                if "NUMBER_LIST START_LIST" in line:
+                    save_events = True
+
+        curry_events = np.array(curry_events, dtype=int)
+
+    else:
+        curry_events = None
+
+    # TODO: This returns events in a curry specific format. This might be reformatted to fit mne
+    return curry_events
+
+
+def _get_curry_version(file_extension):
+    """check out the curry file version"""
+
+    if 'cdt' in file_extension:
+        curry_vers = 8
+
+    else:
+        curry_vers = 7
+
+    return curry_vers
+
+
+def _check_missing_files(full_fname, fname_base, curry_vers):
+    """
+    Check if all neccessary files exist and return the path without extension
+     and its CURRY version
+     """
```

Comment:
check this docstring:
- format
- it says it return but it does not.
- summary missing
- inputs missing (if necessary)